testing/src/main/contraband-scala/sbt/protocol/testing/codec/TestMessageFormats.scala

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
Are you adding the contraband generated files on purpose? I'm not sure this is an oversight or not.